docs/docs/references/cli/fs.md

```diff
@@ -21,19 +21,21 @@ OPTIONS:
    --security-checks value                        comma-separated list of what security issues to detect (vuln,config) (default: "vuln") [$TRIVY_SECURITY_CHECKS]
    --ignorefile value                             specify .trivyignore file (default: ".trivyignore") [$TRIVY_IGNOREFILE]
    --cache-backend value                          cache backend (e.g. redis://localhost:6379) (default: "fs") [$TRIVY_CACHE_BACKEND]
+   --cache-ttl value                              cache TTL when using redis as cache backend (default: 0s) [$TRIVY_CACHE_TTL]
```

Comment:
Since TTL is not supported by BoltDB, only Redis, I'm wondering if it makes sense to rename this flag to --cache-redis-ttl ?

Reply:
At the beginning, I named the flag `--cache-redis-ttl` as you suggested, but I saw BadgerDB supports TTL.
https://github.com/dgraph-io/badger

BadgerDB has almost the same interface as BoltDB and is actively maintained.. We may want to move to BadgerDB in the long term. Then, I renamed `--cache-redis-ttl` to `--cache-ttl` for the future.

But we can start with `--cache-redis-ttl` and rename it later. What do you think?

Reply:
Got it. So let's keep --cache-ttl and maybe print warning if you use it along with the fs backend.

Reply:
Good idea. I'll work on it.